src/neo/Wallets/Wallet.cs

```diff
@@ -298,6 +301,32 @@ public Transaction MakeTransaction(TransferOutput[] outputs, UInt160 from = null
                                 }
                                 sb.EmitAppCall(output.AssetId, "transfer", account, output.ScriptHash, value);
```

Comment:
Note that "value" has two meanings here:
- The "value" being transferred, which is different from balance (good for OpCode.GEQ operations)
- The actual balance from the account, that must be previously taken from the wallet system (good for OpCode.EQUAL operations).

Maybe it's nice to proceed from this point with two values: "transferValue" and "balanceValue".